music/src/core/visualizer.ts

```diff
@@ -437,7 +453,8 @@ export class PianoRollSVGVisualizer extends BaseVisualizer {
   }
 
   private drawNote(
-      x: number, y: number, w: number, h: number, fill: string, index: number) {
+      x: number, y: number, w: number, h: number, fill: string,
+      dataAttributes: DataAttribute[]) {
```

Comment:
actually, i just double checked and I think you could do [{x: number, y: any}] btw, which seems a bit better? Sorrrrrrrry :(

Reply:
Do you mean something like this? I was not able to find what the curly and square bracket thing meant.

Reply:
I thought [{x:number, y:number}] would say that it's an array of objects of that form, but maybe im wrong!